iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/write/DeleteDataNode.java

```diff
@@ -60,6 +60,8 @@ public class DeleteDataNode extends WritePlanNode implements WALEntryValue {
   private final long deleteStartTime;
   private final long deleteEndTime;
 
+  private final boolean isDeleteTimeseries;
```

Comment:
add some comments here to explain why this field doesn't need to be serialized (because it will always be false except for that in delete timesereis, and in which case we will set this to true in `DataNodeInternalRPCServiceImpl.deleteDataForDeleteSchema`.

```suggestion
  private final transient boolean isDeleteTimeseries;
```